README.md

```diff
@@ -92,16 +114,21 @@ After you're finished please remove all the comments and instructions!
     <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
   </ul>
 </details>
+>>>>>>> main
 
 <!-- Features -->
 
 ### Key Features <a name="key-features"></a>
 
 > Describe between 1-3 key features of the application.
 
+<<<<<<< HEAD
+- **[A simple project]**
+=======
 - **[key_feature_1]**
 - **[key_feature_2]**
 - **[key_feature_3]**
+>>>>>>> main
```

Comment:
- [ ] First, remove the merge indicators on lines `125, 127 and 131`.
- [ ] Secondly, please add three features your project includes